lms/services/http.py

```diff
@@ -7,7 +7,7 @@ class HTTPService:
     """Send HTTP requests with `requests` and receive the responses."""
 
     # This is here mostly to let auto-spec know about it in the tests
-    session: Session = None  # typing: ignore
+    session: Session = None  # type: ignore
```

Comment:
This is not supposed to be None, it's a hack for the test, that why we do the ignore instead of fixing the type.